tests/dep-time-travel.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# expected: ESLINT_VERSION numeric env var
+
+npm install --no-save eslint@$ESLINT_VERSION --ignore-scripts || true
```

Comment:
why ignore scripts?

Reply:
I don’t remember, I just copied it from the Travis yaml as it was there. Maybe the old versions have prepublish?